src/AES.cpp

```diff
@@ -1,24 +1,22 @@
 #include "AES.h"
```

Comment:
looks nice but there should be changes in tests and README

Reply:
thx!
ok. I'll update the tests and README.

Reply:
@SergeyBel, I applied the fix. Please confirm~